app.json

```diff
@@ -1,20 +1,18 @@
 {
   "expo": {
+    "name": "insulince",
+    "slug": "insulince",
+    "description": "Insulince is a project aiming to help anyone with Insulin Resistance Syndrome to track nutrition and food intake. https://github.com/rmurphy178/insulince",
+    "version": "0.1.0",
     "sdkVersion": "19.0.0",
     "ios": {
-      "bundleIdentifier": "insulince"
+      "bundleIdentifier": "com.app.insulince"
     },
     "android": {
-      "package": "insulince"
+      "package": "com.app.insulince"
     },
-    "name": "insulince",
-    "slug": "insulince",
-    "version": "0.1.0",
-    "isDetached": true,
-    "detach": {
-      "scheme": "exp72b9b3955440434da72cf9634adb0315",
-      "iosExpoViewUrl": "https://s3.amazonaws.com/exp-exponent-view-code/ios-v1.18.2-sdk19.0.0-3fabbbb7-61b3-4a1e-9932-12d0aa6af276.tar.gz",
-      "androidExpoViewUrl": "https://s3.amazonaws.com/exp-exponent-view-code/android-v1.18.4-sdk19.0.0-3fb414c9-8696-4966-a0ac-9ca424242424.tar.gz"
+    "packagerOpts": {
+      "assetExts": ["ttf"]
```

Comment:
the packager needs to be told that ttf is a valid asset extension (using it for icon files)